src/tools/pub/cmd/pub-token.md

```diff
@@ -0,0 +1,71 @@
+---
+title: dart pub token
+description: Manage authentication tokens for hosted pub repositories.
+---
+
+_pub_ _token_ is one of the subcommands of the [pub command](/tools/pub/cmd).
+
+It is used to update a store of credential tokens used for authenticating
+against third-party servers when [publishing](pub-lish) a package and when
+[retrieving](pub-get) dependencies.
+
+It has three subcommands: `add`, `list` and `remove`.
+
+If you try to `pub get` and have a [dependency](/tools/pub/dependencies) hosted
+on a private repository you will be asked to provide credentials:
+```terminal
+$ dart pub get
+Resolving dependencies... 
+https://some-package-server.com/my-org/my-repo package repository requested authentication! You can provide credential using:
+    pub token add https://some-package-server.com/my-org/my-repo
+
+Go to https://some-package-server.com and log in to obtain your token. 
+```
+
+The last line is a message the server can provide to help you obtaining a token.
```

Comment:
Maybe omit that here... and omit the last line..

-------------------

Or clarify that some servers may not provide such help message.